conductor-client/src/main/kotlin/com/openlattice/data/storage/partitions/PartitionManager.kt

```diff
@@ -58,7 +58,10 @@ class PartitionManager @JvmOverloads constructor(
     }
 
     fun getDefaultPartitions(organizationId: UUID): List<Int> {
-        return organizations.executeOnKey(organizationId, OrganizationReadEntryProcessor { DelegatedIntList(it.partitions) }) as DelegatedIntList
+        val partitions = organizations.executeOnKey(organizationId, OrganizationReadEntryProcessor { DelegatedIntList(it.partitions) })
+        checkNotNull(partitions, {"default partitions for organization $organizationId should not be null"})
```

Comment:
Doesn't kotlin allow checkNotNull(...) {...}? Minor style change as template is only evaluated on null either way.

Reply:
ah, yes
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/check-not-null.html